ArduPlane/Plane.h

```diff
@@ -785,6 +785,9 @@ class Plane : public AP_Vehicle {
     // soaring mode-change timer
     uint32_t soaring_mode_timer;
 
+    // terrain disable for non AUTO modes
+    bool nonauto_terrain_disable;
```

Comment:
none_auto_terrain_disable maybe, its longer but easier to read.

Reply:
okay